Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -17706,15 +18110,22 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = A1EDD2D91884ACE000BBFE98 /* All.xcconfig */;
 			buildSettings = {
+				CLANG_ENABLE_MODULES = YES;
 				PRODUCT_NAME = All;
+				SWIFT_OBJC_BRIDGING_HEADER = "Shared/AuxiliaryProcessExtensions/Framework, XPC Services, Extensions, and daemons-Bridging-Header.h";
+				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_VERSION = 5.0;
 			};
 			name = Debug;
 		};
 		1A50DB3A110A3C13000D3FE5 /* Release */ = {
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = A1EDD2D91884ACE000BBFE98 /* All.xcconfig */;
 			buildSettings = {
+				CLANG_ENABLE_MODULES = YES;
 				PRODUCT_NAME = All;
+				SWIFT_OBJC_BRIDGING_HEADER = "Shared/AuxiliaryProcessExtensions/Framework, XPC Services, Extensions, and daemons-Bridging-Header.h";
+				SWIFT_VERSION = 5.0;
 			};
```

Comment:
Be sure to remove these, and put all settings overrides in xcconfigs. Xcode adds some settings here by default when you create a new target, but we don't want them.

Ditto for the other `XCBuildConfiguration` objects below this.

Reply:
Fixed!

Thanks for reviewing!